workflows/javascript/sdk/order-processor/appWithDaprServer.ts

```diff
@@ -0,0 +1,86 @@
+import { DaprWorkflowClient, WorkflowRuntime, DaprClient } from "@dapr/dapr";
+import { InventoryItem, OrderPayload } from "./model";
+import { notifyActivity, orderProcessingWorkflow, processPaymentActivity, requestApprovalActivity, reserveInventoryActivity, updateInventoryActivity } from "./orderProcessingWorkflow";
+import { DaprServer, CommunicationProtocolEnum } from "@dapr/dapr";
+import express from "express";
+
+const daprHost = "127.0.0.1"; // Dapr Sidecar Host
```

Comment:
What we do in most cases is the following with env var.

```suggestion
const DAPR_HOST = process.env.DAPR_HOST ?? "localhost" // Dapr Sidecar Host
```